src/pb-search.js

```diff
@@ -109,8 +121,9 @@ export class PbSearch extends pbMixin(LitElement) {
         );
         waitOnce('pb-page-ready', (options) => {
             const loader = this.shadowRoot.getElementById('autocompleteLoader');
+            const url = this.source ?? "api/search/autocomplete";
```

Comment:
Using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here will allow the empty string to be used `""`.
If this happens to be set as the default the behaviour might not be the expected one.
We should consider logical or instead (`||`). That way any false value including the empty string will select the default URL.

Reply:
You're right, my bad. I'm just copy-and-paste expert in JavaScript.